content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -92,7 +92,7 @@ To prepare applications for migration from Kubernetes to Docker, address the fol
 |-----------|-------------|
 | Outdated Applications | Update all applications to the latest available version in the TrueNAS catalog before migrating. |
 | Host Path ACLs | Users with applications installed on 24.04 using host path volume mounts with **Enable ACL** selected and **ACL Entries** defined in the app configuration, must go to the **Edit *Application*** screen and set the [**Force Flag** checkbox under **ACL Options**](https://www.truenas.com/docs/scale/24.04/scaletutorials/apps/#storage-configuration-settings) before updating to 24.10. This ensures the app fully migrates and does not encounter issues when the mount point has existing data. |
-| Encrypted Root Dataset | Applications do not migrate to 24.10 if the ix-applications dataset is configured on a pool with an encrypted root dataset (see [NAS-131561](https://ixsystems.atlassian.net/browse/NAS-131561)). Relocate installed applications to an unencrypted pool on 24.04 before attempting to upgrade to 24.10. |
+| Encrypted Dataset | TrueNAS shows a warning and explains that applications do not migrate to 24.10 when the ix-applications dataset is configured on a pool with an encrypted dataset (see [NAS-131561](https://ixsystems.atlassian.net/browse/NAS-131561)). Relocate installed applications to an unencrypted pool on 24.04 before attempting to upgrade to 24.10. TrueNAS does attempt to migrate predefined encrypted host mount paths. |
```

Comment:
should this say the hidden ix-apps dataset? if the user upgrades they still have the ix-applications dataset but new installs do not.

Reply:
ix-applications dataset is appropriate here in context; the table is notifying about considerations before attempting to migrate, so the user will only have the ix-applications dataset in this context.